src/main/java/org/craftercms/studio/impl/v1/service/site/SiteServiceImpl.java

```diff
@@ -191,6 +191,7 @@
     protected SitesServiceInternal sitesServiceInternal;
     protected AuditServiceInternal auditServiceInternal;
     protected ConfigurationService configurationService;
+    @Autowired
```

Comment:
I think we don't have `@Autowired` anywhere else in Studio right? In theory the set should work in the same way do you know if this is a specific issue with MyBatis?

Reply:
I removed `@Autowired` 2 weeks ago to align it with other mappers. Apparently that introduced NPE because `siteFeedMapper` is not set for UM execution. I was not able to provide proper order of bean construction with the `depends-on` attribute. `@Autowired` has to remain until we find a proper solution for bean creation order